tests/test_repo.py

```diff
@@ -150,3 +150,6 @@ def test__get_new_version_patch(self):
         self.repo.semver_label = 'patch'
         self.repo._get_new_version()
         self.assertEqual(self.repo.new_version, '1.2.4')
+
+    def test__get_github_client(self):
+        self.assertIsNotNone(self._github)
```

Comment:
This is not testing the method
`self.assertIstNotNone(self.repo.get_github_client())` may